css-layout-classes/styles.css

```diff
@@ -0,0 +1,49 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+}
+
+div.column-right-d {
+  width: 33%;
+}
+
+img {
+  width: 100%;
+}
+
+.column {
+  width: calc(100% / 3);
+}
+
+div.column-first {
+  width: 33%;
+}
+
+div.column-middle {
+  width: 66%;
+}
+
+div.column-two-thirds {
+  width: 66%;
+}
+
+div.column-quarter {
+  width: 25%;
+}
+
+div.column-second-half {
+  width: 25%;
+}
+
+div.column-three-fourths {
+  width: 25%;
+}
+
+div.column-fourth {
+  width: 25%;
+}
```

Comment:
There should be no need for multiple classes with the exact same styling. Make one class with a good name describing its purpose, and then reuse that class.